### PR TITLE
Compress userdata for encryptor instance

### PR DESCRIPTION
**Pull request by bonzotronix**
The encryptor AMI only looks for multi-part MIME userdata in
the case that it finds the userdata to be compressed.  So we'll
compress it, which is probably a good thing to do anyway.

### CODE REVIEW

File: brkt_cli/encrypt_ami.py [hunk 1]
```diff
@@ -544,19 +549,32 @@ def run_encryptor_instance(aws_svc, encryptor_image_id,
...
         udc.add_file(
             '/var/brkt/ami_config/proxy.yaml',
             proxy_config,
             BRKT_FILES_CONTENT_TYPE
         )
-        user_data_string += '\n' + udc.to_mime_text()
+
+    user_data_string = udc.to_mime_text()
+    try:
+        log.info('Compressing userdata...')
```
Comment (bburtin) [inline brkt_cli/encrypt_ami.py]: Seems gratuitous to log here if it's not a long-running operation.

File: brkt_cli/encrypt_ami.py [hunk 2]
```diff
@@ -544,19 +549,32 @@ def run_encryptor_instance(aws_svc, encryptor_image_id,
...
+        out = StringIO()
```
Comment (bburtin) [inline brkt_cli/encrypt_ami.py]: Is StringIO the right thing to use for binary data?

Comment (bburtin) [inline brkt_cli/encrypt_ami.py]: Never mind.  Apparently this is the sanctioned way to do it.  Weird, I expected a byte buffer.

File: brkt_cli/encrypt_ami.py (hunk 1 above)
Comment (bonzotronix) [inline brkt_cli/encrypt_ami.py]: OK, will remove.

**Comment by bonzotronix** [comment]
Replaced by squashed commits in PR#168

**Pull request closed without merging**